pennylane_lightning/_serialize.py

```diff
@@ -111,7 +112,7 @@ def _serialize_hamiltonian(ob, wires_map: dict, use_csingle: bool):
         rtype = np.float64
         hamiltonian_obs = HamiltonianC128
 
-    coeffs = np.array(ob.coeffs).astype(rtype)
+    coeffs = np.array(unwrap(ob.coeffs)).astype(rtype)
```

Comment:
Is there any other updates required for this on the front-end in PL, or is this sufficient for future support?